cachito/workers/requests.py

```diff
@@ -9,6 +9,10 @@
 
 log = logging.getLogger(__name__)
 
+RETRY_ALLOWED_METHODS = frozenset(
+    {"GET", "POST", "PATCH", "PUT", "DELETE", "HEAD", "OPTIONS", "TRACE"}
```

Comment:
Could you add a comment to explain that this is all the default methods + PATCH and POST? Ideally with a link to the docs where the default methods are mentioned

Reply:
Updated.